storage/replica.go

```diff
@@ -1589,8 +1583,8 @@ func (r *Replica) resolveIntents(ctx context.Context, intents []roachpb.Intent,
 					case *roachpb.NotLeaderError:
 					case *roachpb.RangeNotFoundError:
 					default:
-						// TODO(tschottdorf): Does this need to be a panic?
-						panic(fmt.Sprintf("local intent resolution failed with unexpected error: %s", err))
+						log.Warningf("local intent resolution failed with unexpected error: %s", err)
```

Comment:
Does this change imply that you've seen this error in practice now? In general we shouldn't both log and return an error; just pass them up the stack and log them we we can no longer return the error.